lib/galaxy/config/sample/galaxy.yml.sample

```diff
@@ -2392,17 +2392,24 @@ galaxy:
   # commented out line below).
   #amqp_internal_connection: sqlalchemy+sqlite:///./database/control.sqlite?isolation_level=IMMEDIATE
 
+  # Configuration options passed to Celery.
+  # To refer to a task by name, use the template `galaxy.foo` where
+  # `foo` is the function name of the task defined in the
+  # galaxy.celery.tasks module.
+  # The `broker_url` option, if unset, defaults to the value of
+  # `amqp_internal_connection`.
+  # For details, see Celery documentation at
+  # https://docs.celeryq.dev/en/stable/userguide/configuration.html.
+  #celery_conf:
+    task_routes:
```

Comment:
I'm not seeing where the internal tasks are routed ?

Reply:
We set `"galaxy.internal"` as the `task_default_queue` in the celery module - so anything not explicitly routed falls back to it. (I've also tested this extensively by launching multiple workers, each listening to its own queue: tasks are handled by appropriate workers as expected)